server/tests/src/test/java/org/infinispan/server/functional/AbstractMultiClusterIT.java

```diff
@@ -81,9 +79,8 @@ protected int getCacheSize(String cacheName, RestClient restClient) {
    }
 
    protected void addSchema(RestClient client) {
-      RestCacheClient cache = client.cache(PROTOBUF_METADATA_CACHE_NAME);
-      assertStatus(NO_CONTENT, cache.put("schema.proto", "/* @Indexed */ message Person {  /* @Text(projectable = true) */ required string name = 1; }"));
-      assertStatus(NOT_FOUND,  client.cache(PROTOBUF_METADATA_CACHE_NAME).get("schema.proto.errors"));
+      assertStatus(OK, client.schemas().put("schema.proto", "/* @Indexed */ message Person {  /* @Text(projectable = true) */ required string name = 1; }"));
+      assertStatusAndBodyContains(OK, "pepe", client.schemas().names());
```

Comment:
I know this was intended to fail, but the response correctly contains:
`{"name":"schema.proto","error":null}`
so just check for that